_schemaV2.graphql

```diff
@@ -9331,6 +9351,10 @@ type LotStanding {
   saleArtwork: SaleArtwork
 }
 
+type ManualPayment {
+  isManualPayment: Boolean!
+}
```

Comment:
This is a new, manually-defined type that only exists as a sentinel value. There is no manual payment information to add yet.

Reply:
See my other comment [here](https://github.com/artsy/metaphysics/pull/4086#discussion_r891762542) - I wonder if the `ManualPayment` type should have a descriptor inside it for wire vs other future manual payment types. This would be more useful in the literal sense than a property that duplicates the type name.

Reply:
How about we split by the actual payment method and have `isManual` a property in it.
```GraphQL
type WireTransfer {
  isManual: Boolean!
}
```

This makes it clear for clients what exactly the payment method is and we can add more future manual payment methods.